libraries/freertos_plus/standard/freertos_plus_tcp/source/portable/NetworkInterface/Zynq/NetworkInterface.c

```diff
@@ -50,10 +51,13 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 /* Provided memory configured as uncached. */
 #include "uncached_memory.h"
 
-#ifndef	BMSR_LINK_STATUS
-	#define BMSR_LINK_STATUS            0x0004UL
+#ifndef niEMAC_HANDLER_TASK_PRIORITY
+	/* Define the priority of the task prvEMACHandlerTask(). */
+	#define niEMAC_HANDLER_TASK_PRIORITY	configMAX_PRIORITIES - 1
 #endif
 
+#define niBMSR_LINK_STATUS         0x0004uL
```

Comment:
Since [MII is a standard](https://en.wikipedia.org/wiki/Media-independent_interface), we should collect all of the corresponding defines in an mii.h include file and refer to that.  It looks like MII definitions are being defined in every driver which uses them.

Reply:
As stated here below, Xilinx is standing alone because it doesn't use the module `phyHandling.c` ( because it has a 1 Gbps PHY ).